src/probability/measureScript.sml

```diff
@@ -5394,6 +5394,103 @@ val limsup_suminf_indicator_space = store_thm
       Q.EXISTS_TAC `n'` \\
       MATCH_MP_TAC le_trans >> Q.EXISTS_TAC `&n` >> art [] ]);
 
+(***********************)
+(*   Further Results   *)
+(***********************)
+
+(*  These do not require addition simplifier manipulations on my part. It would
+      probably be more appropriate to add these in the proper places above.
+      - Jared Yeager                                                                   *)
+
+val _ = reveal "C";
+
+(*** measure_space Theorems ***)
+
+Theorem measure_space_cong:
+    ∀sp sts mu nu. measure_space (sp,sts,mu) ∧ (∀s. s ∈ sts ⇒ nu s = mu s) ⇒ measure_space (sp,sts,nu)
```

Comment:
How about `|- (∀s. s ∈ sts ⇒ mu s = nu s) ⇒ (measure_space (sp,sts,mu) <=> measure_space (sp,sts,nu))` ? In this way it may be directly used in a rewrite rule.

Reply:
I'll actually probably go so far as the make the sp0 and sts0 on the LHS and sp1 and sts1 on the RHS (maybe '1' and '2' to match convention, maybe just adding prime on the RHS to match congruence rule convenion, still thinking about that).
Actually, I'll rename these rules with "_eq" or some such (because I undoubtedly use them irule), and then make proper congruence rules for measurable, measure_space, and Borel_measurable.

Reply:
If to be used as a congruence in the simplifier, you will need to also have equalities for all the other sub terms (`sp` and `sts`).  If you omit those, the simplifier won't even look at those sub-terms.  (This can be a powerful and useful trick, but is unlikely to be what you want here.)

Reply:
Indeed, hence my comment about renaming the _cong stuff to _eq (when I still want those results for use with irule) and then making actual congruence results with the _cong suffix. That should be in the most recent (4th) commit.